src/electionguard/election.py

```diff
@@ -16,7 +16,7 @@
 class Configuration:
     """Configuration of election to allow edge cases."""
 
-    allow_overvotes: bool = field(default=False)
+    allow_overvotes: bool = field(default=True)
```

Comment:
This setting is not currently passed through. This will eventually be hooked up. It will however require that information from the `Context` is passed through all the encrypt functions. Likely should be it's own issue following the PR. This is exactly how it is working on C++